kasa/cli.py

```diff
@@ -81,22 +82,42 @@ def cli(ctx, host, alias, target, debug, bulb, plug, strip):
 
 
 @cli.command()
-@click.option("--save")
+@click.option("--scrub", is_flag=True)
 @click.pass_context
-def dump_discover(ctx, save):
+def dump_discover(ctx, scrub):
     """Dump discovery information.
 
-    Useful for dumping into a file with `--save` to be added to the test suite.
+    Useful for dumping into a file to be added to the test suite.
     """
     target = ctx.parent.params["target"]
+    keys_to_scrub = [
+        "deviceId",
+        "fwId",
+        "hwId",
+        "oemId",
+        "mac",
+        "latitude_i",
```

Comment:
Could you also add `latitude` and `longitude` to the list, some devices have these.

Reply:
Added to the srub list